wordfreq_builder/wordfreq_builder/cli/merge_counts.py

```diff
@@ -14,7 +14,8 @@ def merge_lists(input_names, output_name):
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
     parser.add_argument('-o', '--output', help='filename to write the output to', default='combined-counts.csv')
```

Comment:
If all the `help` arguments came last and on their own line, this whole section would become nicely PEP-8 compliant.